pom.xml

```diff
@@ -52,8 +52,8 @@
         <jboss.snapshots.repo.id>jboss-snapshots-repository</jboss.snapshots.repo.id>
         <jboss.snapshots.repo.url>https://s01.oss.sonatype.org/content/repositories/snapshots/</jboss.snapshots.repo.url>
 
-        <quarkus.version>3.18.3</quarkus.version>
-        <quarkus.build.version>3.18.3</quarkus.build.version>
+        <quarkus.version>3.19.0.CR1</quarkus.version>
+        <quarkus.build.version>3.19.0.CR1</quarkus.build.version>
```

Comment:
Have you used the `set-quarkus-version.sh` script? I quickly ran it locally and it seems more dependencies need updating.

Reply:
Ahh, I'll use it.